lib/api/objectDelete.js

```diff
@@ -169,8 +170,14 @@ function objectDelete(authInfo, request, log, cb) {
                     result.versionId : versionIdUtils.encode(
                         result.versionId, config.versionIdEncodingType);
             }
+            const versioningSuspended = bucketMD
+                && bucketMD._versioningConfiguration
+                && bucketMD._versioningConfiguration.status === 'Suspended';
+            const hasByteLength = objectMD && objectMD['content-length'];
+            const byteLength = versioningSuspended && hasByteLength ? objectMD['content-length'] : null;
```

Comment:
At https://github.com/scality/cloudserver/pull/4781/files#diff-3e46aeb1940c95fec15aef92052edc1bdd777d5f297b41e53ef0c5c3b456b5caR195 the `content-length` value is parsed to an integer, there might be cases where it's stored as a string likely? In doubt I'd rather do the same.